yajwt/keys_manager/jwt_keys_memory_manager.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict
+from yajwt.entities.jwt_key import JwtKey, JwtKeyType
+from yajwt.jwt_exceptions import JwtKeyNotFound
+from yajwt.keys_manager.jwt_keys_manager import JwtKeysManager
+
+
+class JwtKeysMemoryManager(JwtKeysManager):
+    def __init__(self, keys: Dict[str, JwtKey]):
+        self.__jwt_keys = keys
+
+    def get_private_key(self, team: str) -> JwtKey:
+        if (
+            team not in self.__jwt_keys
+            or self.__jwt_keys[team].key_type is not JwtKeyType.PRIVATE_KEY
+        ):
+            raise JwtKeyNotFound("JwtKey not found for team '{}'".format(team))
```

Comment:
i'd suggest you to replace .format to f-strings, since they're faster and cleaner than .format() (https://stackoverflow.com/a/43123579/4282400)

eg: `raise JwtKeyNotFound(f"JwtKey not found for team '{team}'")`

this also applies to other .format() usages of this PR :)

Reply:
done!